modules/managing-namespace-auto-pruning-policies.adoc

```diff
@@ -191,6 +191,10 @@ image:auto-prune-policies-page.png[Auto-Prune Policies page]
 
 . Select the desired number of tags to keep. By default, this is set at *20* tags. For this example, the number of tags to keep is set at *3*.
 
+. Optional. To prune only tags that match the given regex pattern, click the *Tag pattern* box and select *match*. In the regex box, enter a pattern to match tags against. For example, to automatically prune all `test` tags, enter `^test.*`. To prune a specific image, you can enter `^test1*`. 
```

Comment:
I am not sure if I understand the sentence  `To prune a specific image, you can enter `^test1*` correctly.`,  If we say `a specific image`,  the regular expression should be `^test1$`,  right?

Reply:
I'd write something like, To prune all tag names starting with `test`, enter `^test.*`